### PR TITLE
SDK-2144. [Upgrader] Check if GitLab CE and EE are supported in upgrader

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-2144

### CODE REVIEW

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Prerequisites
+
+1. [Create a GitLab access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)
+
+GitLab access token should have the following repository permissions:
+
+* **api,write_repository** for Spryker Upgrader Service: grants read and write access to the repository to enable the Upgrader to analyze the project and create PRs.
+
+![GitLab access token](//TODO)
```
Comment (sergeyspryker) [inline docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:21]: Missed link

**Review by sergeyspryker — CHANGES_REQUESTED** [review]

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+![Spryker CI Code](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-azure-managed-project.md/spryker_ci_code_page.png)
+
+4. In the **Connect Git repository** pane, for **GIT HOSTING PROVIDER**, select **Private Server**.
+
+5. For **AUTHENTICATION METHOD** select **Password**.
+
+![Spryker CI Code Azure](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-azure-managed-project.md/azure_code_add.png)
+
+6. Enter your GitLab repository URL, username and password, and click on **Connect repository** button.
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:40]: The Gitlab CE/EE is not working now 

Manual fetch
Today 12:13 PM
integrated: not found.

**Review by sergeyspryker — APPROVED** [review]

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+title: Connect Spryker Code Upgrader to a GitLab self-hosted managed project
+description: Learn how to connect Spryker Code Upgrader to a GitLab self-hosted managed project
+template: howto-guide-template
+redirect_from:
+  - /docs/paas-plus/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-getlab-self-hosted-managed-project.html
+---
+
+## Connect Spryker Code Upgrader using GitLab access token
+
+To connect the Upgrader manually using a Gitlab access token, follow the steps.
```
Comment (AlexSlawinski) [inline docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:11]:
```suggestion
To connect the Upgrader manually using a Gitlab access token, take the following steps.
```

Comment (VladislavStrelchenko) [reply docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:11]: Thanks

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md (hunk 1 above)
Comment (VladislavStrelchenko) [reply docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md]: Fixed

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## To add environment configuration 
+
+1. In Spryker CI, go to **Projects**.
+2. On the **Projects** page, select the **Spryker Upgrade Service** project.
+3. Go to **Pipelines**.
+4. Open **Upgrader** pipline.
+5. Open **Variables** tab.
+
+![Spryker CI Variables Tab](//TODO)
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md]: Yes, fixed

**Review by AlexSlawinski — APPROVED** [review]

**Review by sergeyspryker — DISMISSED** [review]

**Review by DmytroKlymanSpryker — APPROVED** [review]

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-managed-project.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ There are two options for connecting Spryker Code Upgrader to your repository: u
 
 ![Spryker CI GitLab](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-managed-project.md/spryker_ci_gitlab.png)
 
-7. For **GROUP**, select the GitLab group the has access to the repository you want to connect.
+7. For **GROUP**, select the GitLab group of the repository you want to connect.
```
Comment (andriitserkovnyi) [inline docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-managed-project.md:34]:
```suggestion
7. For **GROUP**, select the GitLab group that has access to the repository you want to connect.
```

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## Prerequisites
+
+[Create a GitLab access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)
+
+Your GitLab access token should have the following repository permissions:
+
+* **api** for Spryker CI: grants complete read and write access to the scoped project API, including the Package Registry
+
+* **write_repository** for Spryker Upgrader Service: grants read and write access to the repository to enable the Upgrader to analyze the project and create PRs.
```
Comment (andriitserkovnyi) [inline docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:21]: What's the difference between complete and regular access?

Comment (DmytroKlymanSpryker) [reply docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md:21]: I took this info from official GitLab description:

Screenshot 2023-04-19 at 11 08 01

File: docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-project-self-hosted-with-gitlab.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
+title: Connect Spryker Code Upgrader to a project self-hosted with GitLab
+description: Learn how to connect Spryker Code Upgrader to a project self-hosted with GitLab
+template: howto-guide-template
+---
```
Comment (VadymSachenko) [inline docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-project-self-hosted-with-gitlab.md:5]:
```suggestion
last_updated: Apr 19, 2023
---
```

**Review by VadymSachenko — COMMENTED** [review]
See a comment

**Review by DmytroKlymanSpryker — APPROVED** [review]

**Pull request merged**